cel/program.go

```diff
@@ -283,3 +289,29 @@ func (gen *progGen) Eval(input interface{}) (ref.Val, *EvalDetails, error) {
 	}
 	return v, det, nil
 }
+
+// Cost implements the Coster interface method.
+func (gen *progGen) Cost() (min, max int64) {
+	p, err := gen.factory(emptyEvalState)
```

Comment:
Could you add one last comment here about why it's okay to be empty?

Reply:
done